client/src/containers/event/EventDetailContainer.js

```diff
@@ -0,0 +1,22 @@
+import { connect } from "react-redux";
```

Comment:
Can you rename this file `EventDetailSectionContainer` so it's clear what component it is a container of?